### DISCUSSION TITLE
Daily 28/11/2023

### DISCUSSION THREAD

**Discussion by JasonLAGOUTE** [Sinan-Karakaya/R-Type#14] [Daily]
28/11/2023

**Comment by JasonLAGOUTE** [comment]
Yo ! On continue de lire et de comprendre le sujet, plus approfondir le serveur et le réseau.

**Comment by jovanHillion** [comment]
Hello aujourd'hui:
- je prends encore connaissance du sujet 
- j'ajoute les MUST et SHOULD dans une discussion
- je compile et run le projet
- et si j'ai le temps je commence l'issue du scripting en Lua

**Comment by Sinan-Karakaya** [comment]
Aujourd'hui je bosser sur:

- Intégration de vcpkg
- Un peu de CI
- Puis m'organiser avec les autres sur quoi faire

**Comment by lucasdpt** [comment]
Aujourd'hui:

- Répartition tâches serveur
- Lib network
- Tests network/optimisation

**Comment by mlbonniec** [comment]
Aujourd'hui:
* Prise de connaissance du sujet
* Faxer le CONTRIBUTORS.md à Lucas
* Commencer à séparer/répartir les différentes tâches
